bam-reader/bam-reader.iml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think it should be in .gitignore 🤔

Reply:
yeah, there are some other files also which we need to remove. can be done in another pr